lib/next_ls/runtime/sidecar.ex

```diff
@@ -15,14 +15,23 @@ defmodule NextLS.Runtime.Sidecar do
   end
 
   def handle_info({:tracer, payload}, state) do
+    payload = replace_nil(payload, :start_line, 1)
     DB.insert_symbol(state.db, payload)
 
     {:noreply, state}
   end
 
   def handle_info({{:tracer, :reference}, payload}, state) do
+    payload = replace_nil(payload, :start_line, 1)
```

Comment:
i think we can change the insert query to have `COALESCE(?, 0)` instead of this logic here.

also i think it might be start_line and end_line, but i'll double check

Reply:
When I checked this fix locally, it did not work, because the meta simply does not contain `line`. So I moved the logic to the DB. Is that ok? Or do you want a `Keyword.put_new` in the sidecar?

Reply:
db

Reply:
done

Reply:
sorry, i meant use the `COALESCE` sql function in the insert query